frontend-cert/react-projects/react-props/user-stories.md

```diff
@@ -0,0 +1,22 @@
+# --description--
+
+In this lab, you will create a mood board using reusable React components. The CSS has already been provided for you.
```

Comment:
The descriptions should be fleshed out a bit because most won't know what a mood board is